tests/trustyai/conftest.py

```diff
@@ -0,0 +1,31 @@
+from typing import Generator
+
+import pytest
+from ocp_resources.namespace import Namespace
+
+from tests.trustyai.minio import MinioPod, MinioService, MinioSecret
+
+
+@pytest.fixture(scope="class")
+def minio_pod(model_namespace: Namespace) -> Generator[MinioPod]:
+    with MinioPod(namespace=model_namespace.name) as minio_pod:
```

Comment:
instead of using custom classes, have a `model_namespace` that gets the ns name via request param and then have the code from minio.py in each fixture

Reply:
I don't understand this suggestion, can we discuss it further?